### ISSUE TITLE
Is this version included in PouchDB?

### ISSUE THREAD

**Issue by kentr** [brody2consult/cordova-sqlite-ext#12]
The README says that it is, but in the PouchDB project, I only find references to the standard Cordova-sqlite-storage.

For example, in [`run-cordova.sh`](https://github.com/pouchdb/pouchdb/blob/6567bae3d482e37a08111734b1c402d4f46bbf17/bin/run-cordova.sh#L76).

Ultimately, my goal is to use a pre-populated database with PouchDB in Cordova.

**Comment by brody2consult** [comment]
> The README says that it is, but in the PouchDB project, I only find references to the standard Cordova-sqlite-storage.

No, you have to install the plugin separately. I will add a note when I get a chance.
